src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -1240,7 +1240,7 @@ void Pet::_LoadAuras(PreparedQueryResult result, uint32 timediff)
                 continue;
 
             // negative effects should continue counting down after logout
-            if (remaintime != -1 && !spellInfo->IsPositive())
+            if (remaintime != -1 && (!spellInfo->IsPositive() || spellInfo->HasAttribute(SPELL_ATTR4_FADES_WHILE_LOGGED_OUT)))
```

Comment:
Can you explain me the behaviour? which issue does it fix?

Reply:
continue counting down of negative effects after logout for pets

Reply:
So sorry but i don't understand, maybe i'm missing something...
The SPELL_ATTR4_FADES_WHILE_LOGGED_OUT should flag all spells that must be fades when logout (is it right?)
In your code instead you're saying: if there's a remain time ...and it's a debuff and must be fade, then continue to subtract the remaintime...instead it should be zeroed.
So, i've to ask: have you tested it? Am i missing something? Shouldn't be:
 ! spellInfo->HasAttribute(SPELL_ATTR4_FADES_WHILE_LOGGED_OUT)
?

Reply:
i don't understand what you not understand sorry bad eng.
no this should be:
`(!spellInfo->IsPositive() || spellInfo->HasAttribute(SPELL_ATTR4_FADES_WHILE_LOGGED_OUT)))`
because this check only for negative effects

Reply:
Ok now it's more clear, well i think the || is good because the attribute should work in both positive and negative way. I'm looking for spells with such attribute now

Reply:
plz add "to be merged" again then :)

Reply:
@Viste removed that check. He said on discord that it's a porting of a fix present in another core, but i would understand what kind of "real" issue is fixing. Have we the evidence of some spells that are bugged because doesn't reduce their time during logout?

Reply:
If it fixes something, then would be nice to include in this PR.

![image](https://user-images.githubusercontent.com/147092/63622160-2717ab00-c5f6-11e9-9eea-3361f8134d31.png)

These should be some spells with that flag, theoretically they should be bugged on AC currently because their time is not reducing...but must be tested before

Reply:
you guys decide